### ISSUE TITLE
Latest available version of converter-gson artifact contains security vulnerability transitively.

### ISSUE THREAD

**Issue by TWiStErRob** [square/retrofit#3928]
What kind of issue is this?

 - [x] Feature Request.

![image](https://github.com/square/retrofit/assets/2906988/68d37a8f-c747-4ef3-8ab6-178caacf5b57)

https://devhub.checkmarx.com/cve-details/CVE-2022-25647/

Since Retrofit's last release 2.9.0 (2020) there have been some dependency updates:
 * [GSON 2.8.5 &rarr; 2.8.7](https://github.com/square/retrofit/commit/817de50b913527c13a61341bfee73138861d3f64)
 * [GSON 2.8.7 &rarr; 2.8.9](https://github.com/square/retrofit/commit/97327e80572cdd02bc25de0faa0d710f52f35898)
 * [GSON 2.8.9 &rarr; 2.9.0](https://github.com/square/retrofit/commit/0d8e5b5e48aea4405660612549528f9d6259836b)
 * [GSON 2.9.0 &rarr; 2.10.1](https://github.com/square/retrofit/commit/7c7f28eb00a703ccebc49f64beca8b425289e85a)

among a ton of other maintenance and fixes, but there was no release of Retrofit.

Are there plans to release a new version?

**Comment by JakeWharton** [comment]
We will release when we release. This library has not been a priority recently, but is otherwise stable and working.

In general, both in the past over the history of the project and in the future, we will _never_ do a release to solely address an issue bump for a transitive dependency. Those come along for the ride with new features and bug fixes within this library.

If you want a version newer than we can provide them then I would recommend a sibling dependency declaration with the newer version. This relies on normal Gradle dependency resolution semantics to resolve to the newer version and allows tools like Renovate to update your version when newer ones are available. If the library provides binary compatibility guarantees, as almost all should within a major version, Retrofit's converters and adapters will continue to work just fine.

**Comment by TWiStErRob** [comment]
> to solely address an issue bump for a transitive dependency

Of course, this is issue is just one tiny reason in addition to the [3 years of maintenance updates and enhancements](https://github.com/square/retrofit/compare/2.9.0...master).

**Comment by JakeWharton** [comment]
Yes this is not the only vulnerability in a transitive dependency. Unfortunately the project isn't quite in a state that I'm comfortable releasing so it requires some time to get back to a releasable state.

**Comment by Lonzak** [comment]
Originally I just came here to confirm that this project is dead but then saw that this is not the case. 
I agree with Rob: 3 years of "bugfixes and maintenance" would and should be quite a reason to make a maintenance release imho. 
Due to supply-chain attacks transitive dependencies become more and more important and critical. And of course this belongs to project maintencance, too. Just look at the projects which were updated due to the bug in log4j...

(I understand the maintainer side that many projects are privately maintained, spare time projects etc. But 3 years...)

We are just trying to say that the community would appreciate a maintenance release from time to time :-)  (where time!=3years)
